src/docsfy/cli/admin.py

```
@@ -0,0 +1,210 @@
+                ]
+            )
+
+        print_table(headers, rows)
+    finally:
+        client.close()
+
+
+@users_app.command("create")
+def users_create(
+    username: str = typer.Argument(help="Username to create"),  # noqa: M511
+    role: str = typer.Option("user", "--role", "-r", help="User role (user, admin)"),  # noqa: M511
```

Ruff (0.15.6)

[warning] Invalid rule code in `# noqa`: M511

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)